tests/stdlib/tioutils.nim

```diff
@@ -0,0 +1,51 @@
+discard """
+  output: '''
+hello1
+hello1
+  '''
+"""
+
+import std/ioutils
+import os
+
+# Dummy filename
```

Comment:
nest all below under a `block` so you each test can be scoped when more procs are added eg
```nim
block: # duplicateTo, duplicate
  ...

block: # other proc
```